spec/datadog/core/remote/client/capabilities_spec.rb

```diff
@@ -77,9 +76,24 @@
 
       describe '#base64_capabilities' do
         it 'returns binary capabilities' do
-          expect(capabilities.base64_capabilities).to eq('/A==')
+          expect(capabilities.base64_capabilities).to_not be_empty
         end
       end
     end
   end
+
+  describe '#capabilities_to_base64' do
```

Comment:
Decouple this test from any integration just to ensure every time we add a new remote capability on any component, we do not have to change the test 😄